src/foam/nanos/app/AppBadgeView.js

```diff
@@ -35,22 +35,24 @@ foam.CLASS({
     justify-content: center;
     flex-direction: column;
     gap: 0.8rem;
-    margin: 0 1rem;
     text-align: center;
     font-size: 0.8rem;
     width: 100%;
   }
   ^legal-container{
-    position: absolute;
-    bottom: 1.2rem;
-    left: 0;
+    position: relative;
```

Comment:
Isnt this just default behaviour in this case?

Reply:
Yeah sec, I think I also messed up the popup absolute